### PR TITLE
Fix spin delay issue if mounts with spinning=true

### PR DESCRIPTION
**Pull request by dreamerblue**
* [x] Make sure that you propose PR to right branch: bugfix for `master`, feature for latest active branch `feature-x.x`.
* [x] Make sure that you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
* [x] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
* [x] Rebase before creating a PR to keep commit history clear.
* [x] Add some descriptions and refer relative issues for you PR.

Extra checklist:

**if** *isBugFix* **:**

  * [x] Make sure that you add at least one unit test for the bug which you had fixed.

If a spin component mounts with spinning=true and delay (e.g. in case of rendering a component with data loading initially), it won't be delayed. This PR just fix for it.

### CODE REVIEW

**Comment by yesmeck** [comment]
@dreamerblue Could you add a test case?

**Comment by dreamerblue** [comment]
@yesmeck OK, I have added a test case.

File: components/spin/index.tsx [hunk 1]
```diff
@@ -56,6 +56,14 @@ export default class Spin extends React.Component<SpinProps, SpinState> {
     return !!(this.props && this.props.children);
   }
 
+  componentWillMount() {
```
Comment (yesmeck) [inline components/spin/index.tsx]: Can we use `componentDidMount`? Because `componentWillMount` will be marked as unsafe in the feature versions of react.

Comment (dreamerblue) [reply components/spin/index.tsx]: Forgot to reply there. The method has been changed to "componentDidMount".

**Comment by yesmeck** [comment]
CI failed

**Pull request merged**